tpv5/generating_mirror_mesh.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+prefix=tpv5_f200_halfdom
+# Generate half a mesh using gmsh
+#for gmsh, see http://gmsh.info/#Download
+#gmsh -3 $prefix.geo
+
+# Convert the mesh from neu to hdf5 using pumgen
+# for pumgen, see https://github.com/SeisSol/PUMGen/wiki/How-to-compile-PUMGen
+pumgen $prefix.msh -s msh2
+
+#mirror the mesh using mirrorMesh.py (from here https://github.com/SeisSol/Meshing/tree/master/mirrorMesh/mirrorMesh.py)
+python mirrorMesh.py $prefix.xdmf 1 0
```

Comment:
Line 6 should be uncommented and I would use python3 in line 13. Apart from that it looks fine, I generated a mesh and everything worked well

Reply:
Fixed. Thanks Nico!